mcux/hal_nxp.cmake

```diff
@@ -129,6 +129,7 @@ include_driver_ifdef(CONFIG_COUNTER_MCUX_PIT		pit		driver_pit)
 include_driver_ifdef(CONFIG_COUNTER_MCUX_QTMR		qtmr_1		driver_qtmr_1)
 include_driver_ifdef(CONFIG_DISPLAY_MCUX_ELCDIF		elcdif		driver_elcdif)
 include_driver_ifdef(CONFIG_DISPLAY_MCUX_DCNANO_LCDIF	lcdif		driver_lcdif)
+include_driver_ifdef(CONFIG_DMA_MCUX_PXP		pxp		driver_pxp)
```

Comment:
Since this driver isn't currently integrated into Zephyr as DMA driver could we use `CONFIG_MCUX_PXP` as the Kconfig name? This keeps the Kconfig generic enough for future uses.

Reply:
IMHO that has to be decided by the maintainers if it need to match to a Zephyr subsystem. I can change it as you wish.

Reply:
@dariis could you please help make the change.